application-analytics-ui/src/main/resources/Analytics/Code/AnalyticsTranslations.xml

```diff
@@ -190,7 +190,7 @@ rendering.macro.screenResolution.parameter.startDate.name=End date
 rendering.macro.screenResolution.parameter.startDate.description=Start time of the date interval used for this macro.
 
 ## Admin section
-Analytics.Code.ConfigurationClass_authToken.hint=Token generated by Matomo to be capable of using the API to view the data. This token can be generated from Settings -&gt; Personal -&gt; Security -&gt; Create New Token
+Analytics.Code.ConfigurationClass_authToken.hint=Token generated by Matomo to be capable of using the API to view the data. This token can be generated from Settings -&amp;gt; Personal -&amp;gt; Security -&amp;gt; Create New Token
```

Comment:
I'm not sure why is this change needed

Reply:
It is not necessary, but somehow I double-encoded the text